### ISSUE TITLE
libsvm : error in JDL Arguments

### ISSUE THREAD

**Issue by brucellino** [AAROC/CODE-RADE#69]
The JDL arguments in the libsvm JDL are being interpreted numerically, it seems : 

```
./run_libsvm_rbf.sh: line 119: fastrepo 2K: value too great for base (error token is "2K")
./run_libsvm_rbf.sh: line 121: fastrepo 2K: value too great for base (error token is "2K")
```

**Comment by brucellino** [comment]
the bash manual has the description : 

> Constants with a leading 0 are interpreted as octal numbers.  A leading 0x or 0X 
> denotes hexadecimal.  Otherwise, numbers take the form [base#]n, where base is a 
> decimal number between 2 and 64 representing the arithmetic base, and n is a 
> number in that base.  If base# is omitted, then base 10 is used.
> The  digits  greater than 9 are represented by the lowercase letters, the uppercase 
> letters, @, and _, in that order.  If base is less than or equal to 36, lowercase and 
> uppercase letters may be used interchangeably to represent numbers between 10 and 
> 35.

So "2K" is being interpreted as base2.

Need to find a way for this _not_ to be interpreted as a number, but as a char

**Comment by brucellino** [comment]
fixed in fb4159e8e
